enkit/outputs/commands.go

```diff
@@ -220,7 +220,14 @@ func NewUnmount(root *Root) *Unmount {
 }
 
 func (c *Unmount) Run(cmd *cobra.Command, args []string) error {
-	return fmt.Errorf("`enkit outputs unmount` is unimplemented")
+	if len(args) != 1 {
```

Comment:
Other commands take an invocation_id flag (I think) - can you make sure this command matches the others as much as possible?

Reply:
Done, with -i flag